lib/Viroverse/Controller/Sequence.pm

```diff
@@ -77,6 +77,22 @@ sub show : Chained('load') PathPart('') Args(0) {
     $c->detach( $c->view("NG") );
 }
 
+sub delete : Chained('base') PathPart('delete') Args {
```

Comment:
1. Chain this off of `load` rather than `base` and you won't have to repeat the code for retrieving the sequence; it'll be already available as `$c->model`.
2. The `DELETE` attribute (cf. `POST` elsewhere) will limit the route to matching the given HTTP method. If you do this, you can use an empty `PathPart` and the matched route becomes `DELETE /sequence/:id`. REST!